pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+sorting reports](#filtering-and-sorting-reports).
+
+Once this is done, you can add a report to your shortlist by clicking on the star beside it. The star
+will turn green to indicate that it has been added. Click the stars by any report which you wish to
+investigate.
+
+At a certain level of zoom, there’s also the ability to ‘add all reports visible to shortlist’, which
+allows you to make a bulk addition of everything within the area.
+
+Alternatively, you may add a report from within the individual report page.
+Adding a report to your own list will remove it from any other staff member’s list to which it has
+previously been added.
+
+<img alt="Add a report to your shortlist from within the individual report page" src="/assets/img/pro-user-guide/report-page-shortlist.png" class="pro-screenshot" />
+
+#### Viewing, editing and sorting the shortlist
+
+Click on ‘Shortlist’ in the top menu bar and you will see all the reports which you’ve selected.
+You can continue to add more reports by returning to the ‘all reports’ page.
+
+<img alt="View your shortlist from the link in the navigation bar" src="/assets/img/pro-user-guide/your-shortlist.png" class="pro-screenshot" />
+
+
+If you wish to remove a report from your list, click on the green star again. This can be done on the
+individual report’s page, from the ‘all reports’ page, or from within your shortlist.
+
+Shortlists can be ordered and filtered in the same way as the main list, by using the dropdown
+menus. You can also use the arrows beside each report title to move them up or down the list, until
+they are in a practical order for your route or priorities.
+
+#### Viewing navigation routes
+
+From any report, you can click the button marked ‘navigate to this problem’. This will open a
+separate tab in your browser, and plot a route on Google maps from your current location.
+
+</div>
+
+<div class="pro-task" markdown="1" id="viewing-editing-reports-offline">
+
+### Viewing and editing reports offline
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff and ‘manage shortlist’ must be ticked.</span>
+
+Sometimes it’s useful to be able to see and update reports even when you’re offline — perhaps in
+areas with low mobile connectivity, or because you want to save data. FixMyStreet Pro will let you
+do this, and then it syncs everything up when you’re back online.
+
+Visit your shortlist page before you go out and about: you’ll see a message that the list is being
+cached for use offline. That means that whenever you are viewing the page without an internet
+connection, you’ll be able to see the reports — make sure you have the shortlist page bookmarked
+so that you can find it when you need it.
+
+While offline, you can view reports, create and edit updates. The changes will all go live once you
+go online to sync.
+
+#### Syncing
+
+Your edits will automatically synchronise once you’re back online: just visit FixMyStreet while
```

Comment:
No, you have to press a button to confirm.

Reply:
I don't think I've seen this in action - anywhere I can view it? Or enough just to replace with 'to synchronise your records once you're back online, just visit [which page?] and click [which button?]'

Reply:
Demo site, log in as inspector, it'll take you to shortlist page and cache offline, go offline, edit a report, go online? :) The banner should appear on every page if there's something to be submitted or you're offline.

Reply:
>To synchronise your records once you're back online, just visit any page on FixMyStreet and click the prompt to submit your changes.

(I couldn't actually make it happen via my phone)

Reply:
Updated